.travis.yml

```diff
@@ -75,15 +75,13 @@ script:
   - py.test -vs --cov devito tests/
 
   # Additional seismic operator tests
-  - if [[ $DEVITO_BACKEND != 'yask' ]]; then
-      DEVITO_BACKEND=foreign py.test -vs tests/test_operator.py -k TestForeign;
-      python examples/seismic/benchmark.py test -P tti -so 4 -a -d 20 20 20 -n 5;
-      python examples/seismic/benchmark.py test -P acoustic -a;
-      python examples/seismic/acoustic/acoustic_example.py --full;
-      python examples/seismic/acoustic/acoustic_example.py --constant --full;
-      python examples/seismic/acoustic/gradient_example.py;
-      python examples/checkpointing/checkpointing_example.py;
-    fi
+  - if [[ $DEVITO_BACKEND != 'yask' ]]; then DEVITO_BACKEND=foreign py.test -vs tests/test_operator.py -k TestForeign; fi
```

Comment:
I understand you're not a bash friend (me neither), but can you put all these statements within the same scope? Otherwise this is a mess

Reply:
I'm not sure I understand this right, but wasn't the scoping the reason for the silent failures?  If that is not the type of scoping you mean, can you please provide an example?

Also, I'm not sure I agree this is a mess, since explicit example runs should be used sparingly at best. As such, an individual invocation per example doesn't seem too bad to me (personal opinion of course).

Reply:
please, forget my BS comment